Bottom: 7a73067493886183728c10dec9941f32b02b6447
Top:    a077ce4d5a7997d687ff10799376c44b0c85b0db
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 13:45:47 +0000

Refresh of play-with-quad-edge-data-STEP

---

diff --git a/src/hid/step/object3d_step.c b/src/hid/step/object3d_step.c
index 0c3cd6b..2bc6a9a 100644
--- a/src/hid/step/object3d_step.c
+++ b/src/hid/step/object3d_step.c
@@ -2,12 +2,12 @@
 //#include <stdlib.h>
 //#include <stdbool.h>
 
-#include <glib.h>
+//#include <glib.h>
 
 #include "hid/common/quad.h"
-#include "hid/common/vertex3d.h"
-#include "hid/common/appearance.h"
-#include "hid/common/face3d.h"
+//#include "hid/common/vertex3d.h"
+//#include "hid/common/appearance.h"
+//#include "hid/common/face3d.h"
 //#include "hid/common/edge3d.h"
 #include "hid/common/object3d.h"
 //#include "polygon.h"
